CHANGELOG.md

```diff
@@ -1,3 +1,33 @@
+## 0.54.0 (2025-12-09)
+
+### BREAKING CHANGES
+
+* add translations for multiple languages (#916) ([`a2e8b82`](https://github.com/aws-deadline/deadline-cloud/commit/a2e8b8229921e166cfb215c273a96fbb7fe500c0))
+  * Submitter will now appear in another language if the system language is not English. 
+
+* small files in syncInputJobAttachment causes sync cancel (#908) ([`8f5f29b`](https://github.com/aws-deadline/deadline-cloud/commit/8f5f29bfc71eac1ccd595a2e26714c067957c587))
+  * The parameter `processedFiles` is added to `ProgressReportMetadata`.
+
+### Features
+* add telemetry event upon mounting vfs (#939) ([`890387c`](https://github.com/aws-deadline/deadline-cloud/commit/890387c0958447b78e915977e91c48e17fcfd38f))
+* report paths of downloaded files in json summary (#919) ([`e288776`](https://github.com/aws-deadline/deadline-cloud/commit/e288776ce1c378361852f6ab29f32f650d901f60))
+* add foundations for future non-english UI translations (#901) ([`70dd775`](https://github.com/aws-deadline/deadline-cloud/commit/70dd775b258eb237c9c4c82197fddd006a7887c7))
+
+### Bug Fixes
+* incorrect zh_CN translation for logout (#941) ([`e271941`](https://github.com/aws-deadline/deadline-cloud/commit/e27194158250b55c96f7d2f035c1c86770d7f7f4))
```

Comment:
```suggestion
```

I think we can exclude this because it's a fix for a bug which was never released?